RingPublishingTracking/src/main/java/com/ringpublishing/tracking/RingPublishingTracking+ContentPerformanceMonitor.kt

```diff
@@ -119,13 +123,14 @@ fun RingPublishingTracking.reportContentPageView(
     contentKeepAliveDataSource: RingPublishingTrackingKeepAliveDataSource,
 )
 {
-	// todo: Implement
-// 	reportDelegate.reportContentPageView(
-// 		contentMetadata,
-// 		contentPageViewSource,
-// 		partiallyReloaded,
-// 		contentKeepAliveDataSource
-// 	)
+	with(eventsReporter)
+	{
+		updatePartiallyReloaded(partiallyReloaded)
+		updateStructurePath(currentStructurePath)
+		updatePublicationUrl(contentMetadata.publicationUrl)
+
+		if (partiallyReloaded) newSecondaryId() else newPrimaryId()
```

Comment:
analogicznie mogłoby być wewnątrz + czemu wyżej nie ma newPrimaryId? to jest pageView + to pageview